packages/ember-views/lib/views/view.js

```diff
@@ -1214,48 +1215,36 @@ var View = CoreView.extend({
     @private
   */
   _applyAttributeBindings: function(buffer, attributeBindings) {
-    var attributeValue;
     var unspecifiedAttributeBindings = this._unspecifiedAttributeBindings = this._unspecifiedAttributeBindings || {};
 
-    forEach(attributeBindings, function(binding) {
-      var split = binding.split(':');
-      var property = split[0];
-      var attributeName = split[1] || property;
+    var binding, split, property, attrName, attrNode, attrValue, value;
+    var i, l;
+    for (i=0, l=attributeBindings.length; i<l; i++) {
+      binding = attributeBindings[i];
+      split = binding.split(':');
+      property = split[0];
+      attrName = split[1] || property;
```

Comment:
`attrName` should be the full name in case of namespaced attributes, rather than `split[1]`.  I believe that would take care of https://github.com/emberjs/ember.js/issues/9298.